cypress-tests/cypress/e2e/configs/Payment/Authorizedotnet.js

```diff
@@ -708,5 +710,100 @@ export const connectorDetails = {
         },
       },
     },
+    UCSZeroAuthMandate: {
+      Request: {
+        amount: 0,
+        confirm: false,
+        currency: "USD",
+        customer_id: "Customer123_UCS",
```

Comment:
these can be randomly generated too by making use of `cypress-tests/cypress/utils/RequestBodyUtils.js`!

fyi, you do not need to do hard code these in the first place! cypress takes care of it.